pykafka/cluster.py

```diff
@@ -402,23 +402,24 @@ def get_group_coordinator(self, consumer_group):
         """
         log.info("Attempting to discover offset manager for consumer group '%s'",
                  consumer_group)
-        for i in range(self._max_connection_retries):
+        max_connection_retries = self._max_connection_retries + 5
```

Comment:
But _max_connection_retries is used in other methods, and I do not know that those cases need more retries. Maybe create a separate _max_offset_mgr_connection_retries value?
This was admittedly hacky code, I wanted to see how travis would react.